src/components/Coins.jsx

```diff
@@ -406,6 +347,74 @@ function Coins() {
           )}
         </div>
       )}
+      <div className="flex my-5 w-fit h-auto bg-button-unselected-search-bar-background rounded-md">
+        <div
+          className={`${
+            numOfDays === 0 ? "bg-button-selected" : "bg-transparent"
+          } w-16 h-10 mx-1 my-1 flex justify-center items-center rounded-md font-space-grotesk cursor-pointer`}
+          onClick={() => {
+            setNumOfDays(0);
+          }}
+        >
+          1D
+        </div>
+        <div
+          className={`${
+            numOfDays === 6 ? "bg-button-selected" : "bg-transparent"
+          } w-16 h-10 mx-1 my-1 flex justify-center items-center rounded-md cursor-pointer`}
+          onClick={() => {
+            setNumOfDays(6);
+          }}
+        >
+          7D
+        </div>
+        <div
+          className={`${
+            numOfDays === 30 ? "bg-button-selected" : "bg-transparent"
+          } w-16 h-10 mx-1 my-1  flex justify-center items-center rounded-md cursor-pointer`}
+          onClick={() => {
+            setNumOfDays(30);
+          }}
+        >
+          1M
+        </div>
+        <div
+          className={`${
+            numOfDays === 89 ? "bg-button-selected" : "bg-transparent"
+          } w-16 h-10 mx-1 my-1 flex justify-center items-center rounded-md font-space-grotesk cursor-pointer`}
+          onClick={() => {
+            setNumOfDays(89);
+          }}
+        >
+          90D
+        </div>
+        <div
+          className={`${
+            numOfDays === 179 ? "bg-button-selected" : "bg-transparent"
+          } w-16 h-10 mx-1 my-1 flex justify-center items-center rounded-md font-space-grotesk cursor-pointer`}
+          onClick={() => {
+            setNumOfDays(179);
+          }}
+        >
+          180D
+        </div>
+        <div
+          className={`${
+            numOfDays === 364 ? "bg-button-selected" : "bg-transparent"
+          } w-16 h-10 mx-1 my-1 flex justify-center items-center rounded-md font-space-grotesk cursor-pointer`}
+          onClick={() => {
+            setNumOfDays(364);
+          }}
+        >
+          1Y
+        </div>

```

Comment:
maybe you can put this markup in a component so you don;t repeat yourself so much